From 1fa8a1609c08a35fd60ddc0b2841add789874c36 Mon Sep 17 00:00:00 2001
From: Zhang Sanshan <b51434@freescale.com>
Date: Tue, 29 Dec 2015 16:46:23 +0800
Subject: [PATCH] MA-7426 MA-7426
 android.permission.cts.FileSystemPermissionTest#testDevMemSane

testDevMemSane will check whether /dev/mem exist.If it exist, this case will failed.
/dev/mem is the map of physical address.
CONFIG_DEVMEM is disabled in config of aosp_linux_3.14.

Signed-off-by: Zhang Sanshan <pete.zhang@nxp.com>
---
 arch/arm/configs/imx_v7_android_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/imx_v7_android_defconfig b/arch/arm/configs/imx_v7_android_defconfig
index afecc5b..cd31468 100644
--- a/arch/arm/configs/imx_v7_android_defconfig
+++ b/arch/arm/configs/imx_v7_android_defconfig
@@ -318,6 +318,7 @@ CONFIG_INPUT_ISL29023=y
 CONFIG_INPUT_MPL3115=y
 CONFIG_SENSOR_FXLS8471=y
 # CONFIG_LEGACY_PTYS is not set
+# CONFIG_DEVMEM is not set
 # CONFIG_DEVKMEM is not set
 CONFIG_SERIAL_IMX=y
 CONFIG_SERIAL_IMX_CONSOLE=y
-- 
1.8.0

